hvantk/utils/file_utils.py

```
@@ -0,0 +1,63 @@
+    Download a file from a URL to a local directory.
+
+    :param url: URL of the file to download
+    :param out_dir: Local directory to save the file
+    :param file_name: Name of the file to save
+
+    :return: None
+    """
+import os
+import os.path as path
+
+def download_file(url: str, out_dir: str, file_name: str):
```

Ruff (0.8.2)

Redefinition of unused `download_file` from line 6

(F811)